docs/content/fusd/index.md

```diff
@@ -6,43 +6,53 @@ description: FUSD is a 1:1 USD-backed stablecoin on Flow
 
 Flow USD (FUSD) is a stablecoin on Flow, 
 issued by [Prime Trust](https://www.primetrust.com/), 
-that is backed 1:1 against USD on Flow Mainnet. 
+that is backed 1:1 by the US Dollar.
 
-# Purpose
+## Purpose
 
-FUSD is a 1:1 USD-backed stablecoin that is issued as a fungible 
-token on the Flow network. FUSD is a fixed-price asset that provides 
-a more consistently-priced experience for dapp developers and their users. 
-As a valid payment token, FUSD also acts as an alternative for users 
-who may not be holding the native FLOW token.
+FUSD is a USD-backed stablecoin issued as a fungible 
+token on the Flow network. Its primary purpose is to provide 
+a more consistently-priced payment experience for dapp developers 
+and their users. In general, stablecoins do not demonstrate 
+the volatility that is common amongst other cryptocurrencies, 
+making them a suitable token for everyday trade.
```

Comment:
Maybe we shouldn't imply by suggestion that FLOW is unsuitable? We could remove this line without loosing any important information.